demo/src/content-lists/content-tree-page.js

```diff
@@ -8,85 +8,111 @@ import { parseHtml } from '../core/html-utils';
 import router from '../core/router';
 import { openPlayerModal } from '../player/player-dialog';
 import { contentTreeRootSections } from './content-tree-root-sections';
-
-/**
- * Creates the main content elements for this page : the navigation container and the
- * sections container.
- *
- * @returns {HTMLElement} The created content element.
- */
-const createContentEl = () => {
-  return parseHtml(`
-    <div id="tree-navigation"></div>
-    <div id="sections"></div>
-  `);
-};
+import SpinnerComponent from '../core/spinner-component';
 
 /**
  * Represents the Lists page.
  *
  * @class
  */
 class ListsPage {
+  /**
+   * The spinner component of this page.
+   *
+   * @private
+   * @type {SpinnerComponent}
+   */
+  #spinner;
+  /**
+   * Stack to keep track of the traversal steps for navigation.
+   *
+   * @private
+   * @type {Array<Array<Section>>}
```

Comment:
Isn't it just `Array<Section>`?
```suggestion
   * @type {Array<Section>}
```

Reply:
This is the traversal stack, so it keeps track of whatever levels have been traversed in the tree, each level can have multiple sections (although at the moment only the root element has more than one section).